src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -1285,26 +1369,6 @@ public class C
             await VerifyCSharpAdditionalFileFixAsync(source, shippedText, unshippedText, fixedUnshippedText);
         }
 
-        [Fact]
-        public async Task TestAddAndRemoveMembers_CSharp_Fix_WithAddedNullability_WithOblivious()
```

Comment:
:memo: in using the feature, I realized this is not the right experience: the RS0017 (API removed) shouldn't be there, and the RS0016 (API added) should be a RS0036 (API annotated) instead.

So this test was moved. `LegacyAPIWithObliviousMarkerGetsAnnotatedAsNullable` in `AnnotatePublicApiAnalyzerTests.cs` file shows the new behavior, which is to annotate in-place.